apiary.apib

```diff
@@ -5,6 +5,148 @@ HOST: https://operationcode.org/
+
+
++ Response 201 (application/json)
+
+        {
+        token: Bearer Access-Token
+        }
+
++ Response 422 (application/json)
+
+        {
+        errors: "Some error message"
+        }
+
+## Users | Update [/api/v1/users{?education_level,mentor,scholarship_info,employment_status,company_name,company_role,volunteer,branch_of_service,interests}]
+
++ Parameters
+
+  + education_level (string, required) - Users highest level of education in the form of a string
+  + mentor (boolean, required) - Is User requesting mentor in the form of a boolean
+  + scholarship_info (boolean, required) - Is User interested in scholarship info in the form of a boolean
+  + employment_status (string, required) - Users employment status in the form of a string
+  + company_name (string, required) - Users name of employer in the form on a string
+  + company_role (string, required) - Users role at company in the form of a string
+  + volunteer (boolean, required) - Is User wanting to volunteer in the form of a boolean
+  + branch_of_service (string) - Users branch of service in the form of a string
+  + interests (string) - User interests in the form of a string.
+
+### Update user record [PATCH]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
+
+            {            
+            education_level: "College Graduate" ,   
+            mentor: "true",              
+            scholarship_info: "true",  
+            employment_status: "Employed",
+            company_name: "Comcast",
+            company_role: "Developer",      
+            volunteer: "false",         
+            branch_of_service: "Air Force", 
+            interests: "Python, Python, Python, Python"
+            }
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+            first_name: "Mike",
+            last_name: "Johnson",
+            email: "dreams@operationcode.org",     
+            zip: "80020",        
+            password: "hunter2",    
+            mentor: "true",   
+            education_level: "College Graduate",              
+            scholarship_info: "true",  
+            employment_status: "Employed",
+            company_name: "Comcast",
+            company_role: "Developer",      
+            volunteer: "false",         
+            branch_of_service: "Air Force", 
+            interests: "Python, Python, Python, Python"
+            }
+
++ Response 422 (application/json)
+
+    + Body
+
+            {
+            errors: "Some error message"
+            }
+    
+## Users | Verify [/api/v1/users/profile/verify{?verified}]
+
++ Parameters
+
+  + verified (boolean, required) - Users verified veteran status in the form of a boolean
+
+### Modify User record verified status [POST]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token,
+            verified: true
+
+    + Body
+
+            {
+            verified: true
```

Comment:
See above re: `access_token`

Reply:
Removed verified from both header and body, and fixed indentation.